README.md

```diff
@@ -0,0 +1,13 @@
+# Forta Agent CLI Examples
+
+These examples demonstrate the possible use cases of Forta Agents with little to no configuration.
+
+## Running the Examples
+To run these examples, you'll need to:
+
+1. Navigate to the directory of the example you'd like to run in your CLI
+
+2. Run `npm install` to install the required packages
+
+3. After installation, run `npm run start` to execute the example locally
```

Comment:
I would also add here that if they get an error saying `no jsonRpcUrl provided in config`, then they should create a forta.config.json file in the project folder. this would happen if someone has never done any agent development before i.e. never run `forta-agent init` on their machine. i'd also include the bare minimal contents of the config file i.e.
```
{
    "jsonRpcUrl": "https://mainnet.infura.io/v3/YOUR_API_KEY"
}
```